packages/@cdktf/hcl2cdk/lib/index.ts

```diff
@@ -446,19 +444,22 @@ export async function convertProject(
     language,
     providerSchema,
   });
-  const importMainFile = [imports, inputMainFile].join("\n");
-  const outputMainFile = importMainFile.replace(
-    "// define resources here",
-    code
-  );
-
-  const cdktfJson = { ...inputCdktfJson };
-  cdktfJson.terraformProviders = providers;
-  cdktfJson.terraformModules = tfModules;
 
   return {
-    code: prettier.format(outputMainFile, { parser: "babel" }),
-    cdktfJson,
+    code: (inputMainFile: string) => {
+      const importMainFile = [imports, inputMainFile].join("\n");
+      const outputMainFile = importMainFile.replace(
+        "// define resources here",
```

Comment:
Do we have a test that would fail if we ever accidentally change this in our templates (i.e. change the wording?)
The testcase below in this PR brings its own main.ts containing this comment, so that wouldn't fail.
Maybe we could just add a test that checks if the template contains this string?

Reply:
Added a test 👍